src/please.go

```diff
@@ -616,6 +609,21 @@ var buildFunctions = map[string]func() bool{
 	},
 }
 
+// Run "plz watch.."
+// This guy is a bit different than the rest as it gets the function from the other commands to run as well
```

Comment:
It'd be nice if this was not different from the rest... can it not be a function like the others that basically builds then calls Watch to watch stuff?

Reply:
I don't see why that matters? It can still call other functions in this map from inside itself, there's nothing stopping that?